src/index.html

```diff
@@ -15,8 +16,334 @@
       src="scripts/main.js"
       defer
     ></script>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="icon"
+      href="./images/icons/favicon.png"
+    />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Landing Page</h1>
+    <!-- #region header -->
+    <header class="header">
+      <nav class="nav">
+        <a href="#">
```

Comment:
instead of <a href="#">

maybe <a href="/"> to when click on the logo go to home page